packages/lesswrong/lib/routes.ts

```diff
@@ -713,12 +713,22 @@ const forumSpecificRoutes = forumSelect<Route[]>({
       path: '/wiki',
       redirect: () => '/topics/all'
     },
+    // TODO: use taggingNamePluralSetting for both of these
```

Comment:
Just out of interest, why can't we do that straight away?

Reply:
I guess we could!